amgut/lib/data_access/env_management.py

```diff
@@ -132,8 +133,12 @@ def make_settings_table():
 def populate_test_db():
     conn = SQLConnectionHandler()
 
-    with open(POPULATE_FP) as f:
-        conn.execute(f.read())
+    with gzip.open(POPULATE_FP, 'rb') as f:
+        test_db = f.read()
+
+    command = ['psql', '-d', AMGUT_CONFIG.database]
+    proc = Popen(command, stdin=PIPE, stdout=PIPE)
+    proc.communicate(test_db)
```

Comment:
I've look a bit around. The reason what this is the case is because in the pg_dump, there are clauses like this one:
```
COPY survey_question_response_type (survey_question_id, survey_response_type) FROM stdin;
```
and then a list of values - which are given as if they where given form the standard input.

IMO, it will be better if we can use psycopg2, and you can do that by generating the pg_dump passing the following parameter:
```
--column-inserts             dump data as INSERT commands with column names
```